docker-compose.yml

```diff
@@ -4,16 +4,58 @@ version: '3.9'
 # Gazebo or the real hardware must be started separately
 # Gazebo can be started with ./scripts/start_gazebo.sh
 services:
+  markhor_gazebo:
```

Comment:
So is Gazebo always launched by the compose file?

Reply:
Yes, but you can start only gazebo if you want. I'll add some doc about it, but you can do :

Start only gazebo : docker compose start markhor_gazebo
Start gazebo and slam : docker compose start markhor_gazebo markhor_slam

That way, you don't need a long command or some wizard script. For now, this is only supported on windows with x server, but I'll add support for linux in another pr as suggested by you and sam.